tests/Units/FilterTest.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace Styde\LaravelQueryFilter\Tests\Units;
+
+use Illuminate\Foundation\Testing\RefreshDatabase;
+use Styde\LaravelQueryFilter\Tests\Models\User;
+use Styde\LaravelQueryFilter\Tests\TestCase;
+
+class FilterTest extends TestCase
+{
+    use RefreshDatabase;
+
+    /** @test */
+    public function applying_query_filters()
+    {
+        User::query()->create(['name' => 'Luis', 'email' => 'luis@email.com']);
```

Comment:
`User::create` is sounds better than `User::query()->create()`

Reply:
I use query for IDE autocompletion

Reply:
@luilliarcec me too but in this case it doesn't look good. I'd say if you want to keep the code completion just add a docblock on top of the model. Like we do with facades (i.e.: https://github.com/StydeNet/enlighten/blob/main/src/Facades/Enlighten.php)

Reply:
I'd say we should only use query() when we are writing a long / multiline query, but no for create:

```
User::query()
   ->where...
   ->something...
   ->more_things...
```

We shouldn't tweak our code to please a specific IDEs as other developers will be using other IDEs/Editors/Plugins.

Reply:
You're right, I already made the changes, thanks.